permission_handler/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: permission_handler
 description: Permission plugin for Flutter. This plugin provides a cross-platform (iOS, Android) API to request and check permissions.
 repository: https://github.com/baseflow/flutter-permission-handler
 issue_tracker: https://github.com/Baseflow/flutter-permission-handler/issues
-version: 10.4.2
+version: 10.4.3
```

Comment:
Build fails because this version cannot be found by `pub`, any pointers how you normally deal with this?

Reply:
Build fails because `permission_handler_android: ^10.3.1` cannot be found. This makes sense, as you are updating it in this PR.

We tackle this by creating separate PRs for the different packages. In the case of this PR, you can create a PR for `permission_handler_android`, and a PR for `permission_handler`. We'll merge and publish `permission_handler_android` first, causing the build of `permission_handler` to succeed.

Reply:
I reused this PR to be the PR for the android module, and will create the other one shortly.

Reply:
@JeroenWeener I created the other PR (#1103). I wasn't sure if it should be based on `main` or on my previous PR, so I based it on the PR. Let me know if I should change it.